### DISCUSSION TITLE
Problem after installation

### DISCUSSION THREAD

**Discussion by geofysue** [isce-framework/isce2#486] [Q&A]
Hi, this message error keeps appearing whenever I run `$ISCE_HOME/applications/stripmapApp.py --help` or `obj = isceobj.createImage()`:

`AttributeError: module 'iscesys.ImageApi.DataAccessor' has no attribute 'getTypeSize'`

Does anyone know what's going on? Is my installation correct?

Thanks

**Comment by Basin-Ir** [comment]
I have the same problem. I have tried various methods to resolve it, but so far they have been ineffective. It seems that ISCE2 has not been tested for installation on WSL-Ubuntu, because during compilation some important files are not generated, whereas this issue is related to the VM.
